schema/drive.json

```diff
@@ -6,6 +6,9 @@
   "properties": {
     "defaultRepo": {
       "type": "string", "title": "Default Repository", "default": ""
+    },
+    "gitHubBaseUrl": {
```

Comment:
Maybe just `baseUrl`? I suppose prefixing everything with `github` is a bit redundant in the GitHub Settings.

Reply:
I am fine with either.